Doc/library/pyclbr.rst

```diff
@@ -58,106 +58,112 @@ of these classes.
+      classes.
 
-.. attribute:: Function.is_async
+      .. versionadded:: 3.7
 
-   ``True`` for functions that are defined with the ``async`` prefix, ``False`` otherwise.
 
-   .. versionadded:: 3.10
+   .. attribute:: is_async
+
+      ``True`` for functions that are defined with the ``async`` prefix, ``False`` otherwise.
+
+      .. versionadded:: 3.10
 
 
 .. _pyclbr-class-objects:
 
 Class Objects
 -------------
-Class :class:`Class` instances describe classes defined by class
-statements.  They have the same attributes as Functions and two more.
+
+.. class:: Class
+
+   Class :class:`!Class` instances describe classes defined by class
+   statements.  They have the same attributes as Functions and two more.
 
 
-.. attribute:: Class.file
+   .. attribute:: file
 
-   Name of the file in which the class is defined.
+      Name of the file in which the class is defined.
 
 
-.. attribute:: Class.module
+   .. attribute:: module
 
-   The name of the module defining the class described.
+      The name of the module defining the class described.
 
 
-.. attribute:: Class.name
+   .. attribute:: name
 
-   The name of the class.
+      The name of the class.
 
 
-.. attribute:: Class.lineno
+   .. attribute:: lineno
 
-   The line number in the file where the definition starts.
+      The line number in the file where the definition starts.
 
 
-.. attribute:: Class.parent
+   .. attribute:: parent
 
-   For top-level classes, None.  For nested classes, the parent.
+      For top-level classes, None.  For nested classes, the parent.
 
-   .. versionadded:: 3.7
+      .. versionadded:: 3.7
 
 
-.. attribute:: Class.children
+   .. attribute:: children
 
-   A dictionary mapping names to descriptors for nested functions and
-   classes.
+      A dictionary mapping names to descriptors for nested functions and
+      classes.
 
-   .. versionadded:: 3.7
+      .. versionadded:: 3.7
 
 
-.. attribute:: Class.super
+   .. attribute:: super
 
-   A list of :class:`Class` objects which describe the immediate base
-   classes of the class being described.  Classes which are named as
-   superclasses but which are not discoverable by :func:`readmodule_ex`
-   are listed as a string with the class name instead of as
-   :class:`Class` objects.
+      A list of :class:`!Class` objects which describe the immediate base
+      classes of the class being described.  Classes which are named as
+      superclasses but which are not discoverable by :func:`readmodule_ex`
+      are listed as a string with the class name instead of as
+      :class:`!Class` objects.
 
 
-.. attribute:: Class.methods
+   .. attribute:: methods
 
-   A dictionary mapping method names to line numbers.  This can be
-   derived from the newer children dictionary, but remains for
-   back-compatibility.
+      A dictionary mapping method names to line numbers.  This can be
+      derived from the newer children dictionary, but remains for
```

Comment:
```suggestion
      A :class:`dictionary <dict>` mapping method names to line numbers.
      This can be derived from the newer :attr:`children` dictionary,
      but remains for
```